curvefs/src/metaserver/storage/rocksdb_options.cpp

```diff
@@ -158,12 +179,18 @@ void InitRocksdbOptions(
     defaultCfOptions.enable_blob_files = true;
     defaultCfOptions.level_compaction_dynamic_level_bytes = true;
     defaultCfOptions.compaction_pri = rocksdb::kMinOverlappingRatio;
+    defaultCfOptions.level0_file_num_compaction_trigger =
+        FLAGS_rocksdb_level0_file_num_compaction_trigger;
+    defaultCfOptions.max_bytes_for_level_base =
+        FLAGS_rocksdb_max_bytes_for_level_base;
     defaultCfOptions.prefix_extractor.reset(
         rocksdb::NewFixedPrefixTransform(RocksDBStorage::GetKeyPrefixLength()));
     defaultCfOptions.memtable_prefix_bloom_size_ratio =
         FLAGS_rocksdb_memtable_prefix_bloom_size_ratio;
     defaultCfOptions.table_factory.reset(
         rocksdb::NewBlockBasedTableFactory(tableOptions));
+    defaultCfOptions.table_properties_collector_factories.emplace_back(
+        rocksdb::NewCompactOnDeletionCollectorFactory(10000, 1000));
```

Comment:
I think the thrid argument `deletion_ratio` might be also usefull. And, please don't hard code 10'000 1'000 in here, give them more meaningfull names.

Reply:
> I think the thrid argument `deletion_ratio` might be also usefull. And, please don't hard code 10'000 1'000 in here, give them more meaningfull names.

fixed.